lib/payrix.rb

```diff
@@ -6,7 +6,6 @@
 require 'payrix/configuration'
 require 'payrix/exceptions'
 require 'payrix/http'
-require 'payrix/resource'
```

Comment:
This is the root for all of the old interface. This file, and all cascading files for `Payrix::Resource::...` have been removed below.